Racecar_Lab/src/SolarCar.ts

```diff
@@ -0,0 +1,16 @@
+import { Racer } from "./Racer"
+
+export class SolarCar implements Racer {
+    team: string
+    speed: number
+    constructor(team: string, speed: number = 0) {
+        this.team = team
+        this.speed = speed
+    }
+    accelerate(): void {
```

Comment:
The return value types are annotated in the interface already
```suggestion
    accelerate() {
```